packages/experiments-realm/crm/contact.gts

```diff
@@ -294,7 +312,7 @@ class FittedTemplate extends Component<typeof Contact> {
           </div>
         {{/if}}
 
-        {{#if @model.phoneMobile}}
+        {{#if (formatPhone @model.phoneMobile)}}
           <div class='row primary-phone'>
             <PhoneIcon class='icon gray' />
             <span>{{formatPhone @model.phoneMobile}}</span>
```

Comment:
The formatting and styling for this should be in the view or atom template of this Phone field def.